src/session.ts

```diff
@@ -755,26 +764,22 @@ export class SessionManager implements Middleware {
                 new SessionMenuItem(
                     "Restart Current Session",
                     () => { this.restartSession(); }),
+
+                ...powerShellItems,
```

Comment:
I dislike how easy it is to miss this syntax (especially in a list with two other new objects), but I don't really have a serious complaint. It's idiomatic and I'll come to like it I'm sure.

Reply:
I've added a comment so it's a little bit clearer 🙂